samples/02_power_users_developers/openstreetmap_exploration.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<ul><li>in the environment without <em>ArcPy, </em>I receive a value of -173 Square Kilometers</li><li>I installed the <em>shapely </em>package and that corrected it initially, but now I'm seeing this value after running it again.</li><li>I'm assuming with the correct modules this doesn't occur - did you notice negative values at any time?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>